From d1abeec8d11cdfe6cc9316de4da8f07c1ba295c6 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Thu, 27 Jun 2024 14:37:53 +0800
Subject: [PATCH 33/73] DEEPIN: net: stmmac: fix dwmac-phytium build on 6.9

Declare phytium_dwmac_remove() as a static function to resolve a missing
prototype warning.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>

Link: https://github.com/deepin-community/kernel-rolling/commit/a12ca6cda2b3cfc3239dae8c94da2ac6255b98e7
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
index ba86b6bb0..eea842195 100644
--- a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
@@ -199,7 +199,7 @@ static int phytium_dwmac_probe(struct platform_device *pdev)
 	return  stmmac_dvr_probe(&pdev->dev, plat, &stmmac_res);
 }
 
-int phytium_dwmac_remove(struct platform_device *pdev)
+static int phytium_dwmac_remove(struct platform_device *pdev)
 {
 	struct net_device *ndev = platform_get_drvdata(pdev);
 	struct stmmac_priv *priv = netdev_priv(ndev);
-- 
2.49.0

